src/_includes/zebra/character-count.html

```diff
@@ -0,0 +1,54 @@
+{% comment %}
+  Character count
+
+  **Note:** This component includes changes specifically for zebra testing that diverge from the default component.
+
+  Variables:
+    - required: Boolean to determine if input is required.
+    - input_name: String of the input's name. Used to create the label and hint ID.
+    - label_text: String for the input label.
+    - character_limit: Number for the component character limit count.
+    - hint_text: String for optional input hint text.
+
+  Used in `src/_includes/zebra/forms/book-tour.html`.
+{% endcomment %}
+
+{% assign input_id = input_name | default: "character-count" %}
+{% assign input_label_id = input_id | append: "-label" %}
+
+<div class="usa-character-count">
+  <div class="usa-form-group {% if error_message  %}usa-form-group--error{% endif %}">
+    <label class="usa-label" for="{{ input_id }}">
+      {{ label_text }}
+
+      {% if required %}
+      {% include "required-hint.html" %}
+      {% endif %}
+
+      <svg
+        class="usa-icon usa-tooltip"
+        data-position="top"
+        title="Accommodations are available but limited"
+        role="img">
+          <use href="{{ '/assets/img/sprite.svg#info' | url }}"></use>
+      </svg>
+    </label>
+
+    {% if error_message %}
+      <span class="usa-error-message" role="alert">
+        {{ error_message }}
+      </span>
+    {% endif %}
+
+    <textarea
```

Comment:
Typing into the text box for accommodations the character count jumps from 50 to 45 after I type in one letter

Reply:
Fixed in 367b2ebee5cdb131591d4b58f8db44b797b3c91b.

Reply:
Confirmed with 👍 that it was fixed, resolving thread.